js/app.js

```diff
@@ -237,7 +237,7 @@ app.controller('AltcoinController', function($scope, $interval, Order, AltcoinNe
         else
             params = {};
         params.uuid = uuid;
-        url = window.location.pathname;
+        url = window.location.protocol + "//" + window.location.host + window.location.pathname;
```

Comment:
why not use window.location.href ?

Reply:
We do not require the parameters from the current URL